chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/listing/CertifiedProductDetailsEntity.java

```diff
@@ -259,587 +255,6 @@
-        this.contactId = contactId;
-    }
-
-    public String getFullName() {
-        return fullName;
-    }
-
-    public void setFullName(final String fullName) {
-        this.fullName = fullName;
-    }
-
-    public String getEmail() {
-        return email;
-    }
-
-    public void setEmail(final String email) {
-        this.email = email;
-    }
-
-    public String getPhoneNumber() {
-        return phoneNumber;
-    }
-
-    public void setPhoneNumber(final String phoneNumber) {
-        this.phoneNumber = phoneNumber;
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setTitle(final String title) {
-        this.title = title;
-    }
-
-    public Long getDeveloperStatusId() {
-        return developerStatusId;
-    }
-
-    public void setDeveloperStatusId(final Long developerStatusId) {
-        this.developerStatusId = developerStatusId;
-    }
-
-    public String getDeveloperStatusName() {
-        return developerStatusName;
-    }
-
-    public void setDeveloperStatusName(final String developerStatusName) {
-        this.developerStatusName = developerStatusName;
-    }
-
-    public ProductEntity getProduct() {
-        return product;
-    }
-
-    public void setProduct(final ProductEntity product) {
-        this.product = product;
-    }
-
-    public Integer getCountSurveillance() {
-        return countSurveillance;
-    }
-
-    public void setCountSurveillance(final Integer countSurveillance) {
-        this.countSurveillance = countSurveillance;
-    }
-
-    public Integer getCountOpenSurveillance() {
-        return countOpenSurveillance;
-    }
-
-    public void setCountOpenSurveillance(final Integer countOpenSurveillance) {
-        this.countOpenSurveillance = countOpenSurveillance;
-    }
-
-    public Integer getCountClosedSurveillance() {
-        return countClosedSurveillance;
-    }
-
-    public void setCountClosedSurveillance(final Integer countClosedSurveillance) {
-        this.countClosedSurveillance = countClosedSurveillance;
-    }
-
-    public Integer getCountOpenNonconformities() {
-        return countOpenNonconformities;
-    }
-
-    public void setCountOpenNonconformities(final Integer countOpenNonconformities) {
-        this.countOpenNonconformities = countOpenNonconformities;
-    }
-
-    public Integer getCountClosedNonconformities() {
-        return countClosedNonconformities;
-    }
-
-    public void setCountClosedNonconformities(final Integer countClosedNonconformities) {
-        this.countClosedNonconformities = countClosedNonconformities;
-    }
-
-    public Date getCertificationStatusDate() {
-        return Util.getNewDate(certificationStatusDate);
-    }
-
-    public void setCertificationStatusDate(final Date certificationStatusDate) {
-        this.certificationStatusDate = Util.getNewDate(certificationStatusDate);
-    }
-
-    public Date getDecertificationDate() {
-        return Util.getNewDate(decertificationDate);
```

Comment:
This isn't the default getter. There are a few other Date values here. I don't know if this will work as it is, but it's changing code that I don't know we're expecting